STUDIO/org.openl.rules.webstudio/webapp/pages/modules/administration/users/editUser.xhtml

```diff
@@ -160,25 +160,26 @@
         let editUserIsLastNameExternal;
         let editUserIsDisplayNameExternal;
 
-        function showEditUserForm(username, firstName, lastName, email, displayName, internalUser, userGroups, isSuperuser, isCurrentUser, firstNameExternal, lastNameExternal, emailExternal, displayNameExternal) {
-            editUserIsEmailExternal = emailExternal === true;
-            editUserIsFirstNameExternal = firstNameExternal === true;
-            editUserIsLastNameExternal = lastNameExternal === true;
-            editUserIsDisplayNameExternal = displayNameExternal === true;
+        function showEditUserForm(userModel) {
+            const externalFlags = userModel.externalFlags || {};
+            editUserIsEmailExternal = externalFlags.emailExternal === true;

```

Comment:
Is it necessary to use === here (and in other similar places)?
Can externalFlags be undefined or nullable or contain non-bolean value?

Reply:
To be sure that there is a boolean type.